src/System.Data.SqlClient/src/System/Data/SqlClient/PoolBlockingPeriod.NetCoreApp.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I think it is safe to remove `.NetCoreApp.` from the filename since this file is only compile on NetCoreapp and there is no other `PoolBlockingPeriod.cs`